tests/integration/suite/test_istio.py

```diff
@@ -66,6 +66,45 @@ def test_destination_rule(admin_pc):
     client.delete(ns)
 
 
+@pytest.mark.nonparallel
```

Comment:
Do we need it to be nonparallel?

Reply:
Probably not for a destination rule, for virtual services they were likely testing similar things so they could mess with each other. I'll update.